chain/vm/invoker.go

```diff
@@ -284,7 +284,8 @@ func DecodeParams(b []byte, out interface{}) error {
 }
 
 func DumpActorState(i *ActorRegistry, act *types.Actor, b []byte) (interface{}, error) {
-	if builtin.IsAccountActor(act.Code) { // Account code special case
+	// Account & Embryo code special case
+	if builtin.IsAccountActor(act.Code) || builtin.IsEmbryoActor(act.Code) {
```

Comment:
This shouldn't be necessary, even for the account case.

Reply:
This is gone now.